internal/config/main.go

```diff
@@ -30,6 +30,9 @@ const (
 	defaultAutoDetectFrequency           = 5 * time.Second
 	defaultCollectorConfigMapEntry       = "collector.yaml"
 	defaultTargetAllocatorConfigMapEntry = "targetallocator.yaml"
+	AutoscalingVersionV2                 = "v2"
```

Comment:
nit: define these as enum (a golang type)

Reply:
@pavolloffay Can you point me to an example of what you want here?    Googling "golang enums" returns a bunch of disparate answers